reconcile/utils/gql.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, url, token=None, int_name=None, validate_schemas=False):
             if not self._valid_schemas:
                 raise GqlApiIntegrationNotFound(int_name)
 
-    @retry(exceptions=GqlApiError, max_attempts=5, hook=capture_and_forget)
```

Comment:
why is this change?

Reply:
We will be getting `TypeError: retry() got an unexpected keyword argument 'hook'`

Reply:
Relevant pr: https://github.com/app-sre/qontract-reconcile/pull/1430

Reply:
I think you need to update your sretoolbox https://github.com/app-sre/sretoolbox/commit/010f38cc081ff4e6e22453a33db5bbc65e36a33f

Reply:
ahh I see! Thanks ;)